cfme/tests/containers/test_containers_properties.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+
+""" Field validation for properties table.
+"""
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.containers.pod import Pod, list_tbl as list_tbl_pod
+from cfme.containers.route import Route, list_tbl as list_tbl_route
+from cfme.containers.project import Project, list_tbl as list_tbl_proj
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+
+""" Polarion test case CMP-9911.
+"""
```

Comment:
I believe this belongs as a test docstring (right after `def ...`) rather than being outside.